src/sparse/node.rs

```diff
@@ -201,6 +201,10 @@ impl Node {
 impl crate::common::Node for Node {
     type Key = Bytes32;
 
+    fn height(&self) -> u32 {
+        todo!()
```

Comment:
nit: A more descriptive comment for todo would be nice to have. For instance using a link to a GitHub issue is something I like to do. `todo!("https://example.org/link/to/issue/123")`. This also ensures that we have issues open for all unimplemented functionality.